src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.cpp

```diff
@@ -130,6 +130,9 @@ bool LazyTensorImpl::operator==(const LazyTensorImpl& other) const {
     case TransformType::CONVERT:
         // everything is already compared above - skip
         break;
+    case TransformType::RESHAPE:
+        // everything is already compared above - skip
+        break;
```

Comment:
```suggestion
    case TransformType::THIS:
    case TransformType::CONVERT:
    case TransformType::RESHAPE:
        // everything is already compared above - skip
        break;
```
If clang-format allows such code, ofc.

Reply:
Removed